openmls/src/messages/mod.rs

```diff
@@ -262,11 +263,6 @@ impl GroupInfo {
         &self.payload.confirmation_tag
     }
 
-    /// Returns the signer.
```

Comment:
This is not used anymore in `pub(crate)`. If we want to keep it, we should define a public API.